src/app/zap_cluster_list.json

```
@@ -180,6 +180,8 @@
         ],
         "AIR_QUALITY_CLUSTER": ["air-quality-server"],
         "ALARM_CLUSTER": [],
+        "AMBIENT_CONTEXT_SENSING_CLUSTER": [],
```

Biome (2.5.6)

[error] The key AMBIENT_CONTEXT_SENSING_CLUSTER was already declared.

(lint/suspicious/noDuplicateObjectKeys)